toolbar/core/vite.config.ts

```diff
@@ -22,7 +22,7 @@ export default defineConfig({
     extensions: ['.mjs', '.js', '.ts', '.jsx', '.tsx', '.json'],
   },
   esbuild: {
-    minifyIdentifiers: false,
+    minifyIdentifiers: true,
```

Comment:
There's a **minification error when building the monorepo**, (a shortened variable has a collision). I'll have a look into that! 
<img width="804" alt="image" src="https://github.com/user-attachments/assets/f0e525f2-2132-46e4-a15f-7f3c357bf81a" />